core/src/main/scala/org/apache/spark/api/java/JavaRDDLike.scala

```diff
@@ -131,7 +131,7 @@ trait JavaRDDLike[T, This <: JavaRDDLike[T, This]] extends Serializable {
    */
   def flatMapToDouble(f: DoubleFlatMapFunction[T]): JavaDoubleRDD = {
     def fn: (T) => Iterator[jl.Double] = (x: T) => f.call(x).asScala
-    new JavaDoubleRDD(rdd.flatMap(fn).map((x: jl.Double) => x.doubleValue()))
+    new JavaDoubleRDD(rdd.flatMap(fn).map(x => x.doubleValue()))
```

Comment:
Thanks! I just wanted to be a bit careful because it looks it uses a variable name `x` just above and thought the author might intend to want that variable visible, which is pretty trivial. Let me correct this soon!